js/adapt-contrib-bookmarking.js

```diff
@@ -52,10 +54,10 @@ define([
                 } catch (error) {
                     return;
                 }
-                
+
```

Comment:
No need for try...catch in framework 2.2+.

Not even sure `model` needs to be assigned to a variable?